cmd/generate/config/rules/django.go

```diff
@@ -0,0 +1,26 @@
+package rules
+
+import (
+	"github.com/zricethezav/gitleaks/v8/cmd/generate/config/utils"
+	"github.com/zricethezav/gitleaks/v8/config"
+	"github.com/zricethezav/gitleaks/v8/regexp"
+)
+
+func DjangoInsecureSecret() *config.Rule {
+	// define rule
+	r := config.Rule{
+		RuleID:      "django-insecure-secret-key",
+		Description: "Detected a Django insecure secret key, potentially compromising Django’s security protections against privilege escalation and remote code execution.",
+		Regex:       regexp.MustCompile(`["'](django-insecure-[^"']{1,250})["']`),
+		Path:        regexp.MustCompile(`(?i)\.py$`),
+		SecretGroup: 2,
+		Keywords:    []string{"django"},
```

Comment:
This can be made more specific, since `django-insecure-` is a fixed value.
```suggestion
		Keywords:    []string{"django-insecure-"},
```